cpp/models/sde_seirvv/README.md

```diff
@@ -0,0 +1,24 @@
+
+Below is an overview of the model architecture.
+
+
+
+## Simulation
+
+The simulation runs in discrete time steps using an euler-maruyama integration scheme. The Simulation class handles the parameters and the numerical integrator. It also stores the result. 
+![SEIRVV_model](https://github.com/user-attachments/assets/8bb70557-071e-4ec0-8e5a-81380eb2086c)
+| Mathematical variable                   | C++ variable name | Description |
+|---------------------------- | --------------- | -------------------------------------------------------------------------------------------------- |
+| $\phi$                      |  `ContactPatterns`               | Daily contact rate / Number of daily contacts. |
+| $\rho_1$                      |  `TransmissionProbabilityOnContactV1`               | Transmission risk for people located in the Susceptible compartment (susceptible to infection with variant 1). |
+| $\rho_2$                      |  `TransmissionProbabilityOnContactV2`               | Transmission risk for people located in the Susceptible compartment or in the RecoveredV1 compartment (susceptivle to infection with variant 2). |
+| $N$                         | `populations.get_total()`   | Total population. |
+| $T_{E1}$                    |  `TimeExposed1`               | Average time in days an individual stays in the ExposedV1 compartment. |
+| $T_{E2}$                    |  `TimeExposed2`               | Average time in days an individual stays in the ExposedV2 or in the ExposedV1V2 compartment. |
+| $T_{I1}$                    |  `TimeInfected1`               | Average time in days an individual stays in the InfectedV1 compartment. |
+| $T_{I1}$                    |  `TimeInfected2`               | Average time in days an individual stays in the InfectedV1 or in the InfectedV2 compartment. |
```

Comment:
I have not decided on a convention here yet but you are probably right. I_1 at least is a better variable name than I1

Reply:
Actually I don't think T_{I_{2}} is what I want because that parameter is used both for the compartments with _2 and the compartments with _{1,2}. I will use T_{I,2} instead